templates/zerver/help/send-a-message-in-a-different-language.md

```diff
@@ -0,0 +1,78 @@
+# Send a message in a different language
+
+To send a message in a different language you have to change your keyboard settings
+through your Operating System.
+
+# Change keyboard language on Linux
+
+{go-to-the.md} *System Settings* via the distibution specific application
+launcher.
+
+{go-to-the.md} *Text Entry* page.
+
+3. Add an input language by clicking on the plus
+(<i class="icon-vector-plus"></i>) icon.
+
+4. Select the language you want from the menu that appears.
+
+5. Next, select this language from the icon in your menu bar
+that is currently the abreviation for your current language. 
+This sets your current keyboard language to the selected language.
+
+6. If you would like to revert back to your previous language, reselect
+your previous language from the language selector in the menu bar.
+
+# Change keyboard language on Windows
+
+1. Click Start, type intl.cpl in the *Start Search* box, and then press
+ENTER.
+
+2. On the *Keyboards and Language* tab, click *Change keyboards*.
+
+3. Click *Add*.
+
+4. Expand the language that you want.
+
+5. Expand *Keyboard list*, click to select the checkbox of the language
+that you want, and then click *OK*.
+
+6. In the *options*, click *View Layout* to compare the layout with the
+actual keyboard.
+
+7. In the *Default input language* list, click on your desired language,
+and then click *OK*.
+
+8. In the *Regional and Language Options* dialog box, click *OK*.
+
+!!! tip ""
```

Comment:
This restarts the numbering:
![screen shot 2017-01-13 at 10 45 04 am](https://cloud.githubusercontent.com/assets/641763/21919605/6529aac8-d97d-11e6-80ca-d0837268d7bd.png)

(Ignore this if it works correctly for you.)